docs/DeveloperGuide.md

```diff
@@ -25,6 +25,7 @@ The following websites and codebases were referenced and adapted for our project
 * AddressBook-Level2 project ([Website](https://se-education.org/addressbook-level2/) | 
 [GitHub](https://github.com/se-edu/addressbook-level2))
 * Team Member Alan Low's individual project (iP) codebase ([GitHub](https://github.com/alanlowzies/ip))
+* AddressBook-Level3 project([Website](https://se-education.org/addressbook-level3/DeveloperGuide.html))
```

Comment:
Perhaps this reference can be placed directly below 'AddressBook-Level2 project'? Since the two of them are related.